clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/SubmitJobRequest.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright 2015 Netflix, Inc.
```

Comment:
Ahh ok, that makes sense. I didn't pickup that `titan-client` was killed and moved in here.